test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -98,6 +98,38 @@ public void TestEstimatorSaveLoad()
             Done();
         }
 
+        [Fact]
+        public void TestLoadImages()
+        {
+            var env = new MLContext(1);
+            var dataFile = GetDataPath("images/images.tsv");
+            var correctImageFolder = Path.GetDirectoryName(dataFile);
+            var data = TextLoader.Create(env, new TextLoader.Options()
+            {
+                Columns = new[]
+                {
+                        new TextLoader.Column("ImagePath", DataKind.String, 0),
+                        new TextLoader.Column("Name", DataKind.String, 1),
+                    }
```

Comment:
could you align the brackets?

Reply:
It seems a lot a lot of these brackets for the `Column` field are misaligned. I'll fix them as well, thanks!